package.json

```diff
@@ -34,13 +34,17 @@
   "dependencies": {
     "@emotion/react": "11.7.1",
     "@mui/material": "5.3.1",
+    "@reduxjs/toolkit": "1.7.2",
```

Comment:
Is there a reason to add this to the library?
I don't see it used anywhere
Same for `redux`

Reply:
mistakenly added this because of AssetHeaderBlock. will remove this